server.js

```diff
@@ -26,34 +26,46 @@ var manifest = {
             }
         }
     ],
-
-    plugins: {
-
-        // General porpoise
-        './dogwater':   Config.dogwater,
-        './poop':       Config.poop,
-
-        // Server-specific
-        '../lib': [{ select: 'boilerplate-api' }]
-
-    }
+    registrations: [
+        {
+            plugin: {
+                register: './dogwater',
+                options: Config.dogwater
+            }
+        },
+        {
+            plugin: {
+                register: './poop',
+                options: Config.poop
+            }
+        },
+        {
+            plugin: {
+                register: '../lib',
+                options: [{ select: 'boilerplate-api' }]
```

Comment:
It was probably always wrong, but I think this needs to be moved to be a sibling of the `plugin` property (since it contains registrations options for hapi– not plugin options).  Mind moving that while we're touching the glue config?